todo_app/templates/index.html

```diff
@@ -11,7 +11,11 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+
+        {% for item in items %}
+           <li>{{ item.title }} - {{ item.status }} - {{ item.main }}</li>
+           {% endfor %}
+        
       </ul>
     </div>
```

Comment:
You need to add a form element to pass new to-do items to the POST route I have asked for in the `app.py` file.

https://www.w3schools.com/html/html_forms.asp